src/exports.jl

```diff
@@ -1333,6 +1333,7 @@ export ring_elem_type
 export ring_type
 export rising_factorial
 export root
+export roots
```

Comment:
again, I am not really objecting, just surprised as this seems to go beyond the PR description, so I am double checking to verify this is intended?

Reply:
roots is reexported from Hecke anyway. Oscar.LieAlgebras was kind of inconsistent about exporting it.
And it felt somehow wrong to export it from Oscar in the LieAlgebras experimental folder, although it is not introduced there (as everything else that is exported there) but imported from Oscar/Hecke/...
This felt cleaner than just removing the export from OSCAR entirely and rely on the reexport